label.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'item'
```

Comment:
- [ ] [OPTIONAL] Do the same as I suggested in the `book.rb` file. You can add this file and `book.rb` in the same folder.